README.md

```diff
@@ -106,53 +115,49 @@ database = "telegraf"
  - Default polling interval is 10 seconds, if you want to change that have to add /change the Agent Interval value.
 
 ## Start the telegraf service
-
+Use the commands below to start telegraf:
```

Comment:
"start telegraf and check its status"

Reply:
Thanks! This, and the other 2 comments are resolved now.